util/request.py

```diff
@@ -27,22 +27,25 @@ def crossorigin(anonymous=True):
     def decorate(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
-            cors_origin = app.config.get("CORS_ORIGIN", "*")
+            cors_origins = app.config.get("CORS_ORIGINS", "*")
```

Comment:
if you are taking an input as list then this would be `["*"]` no?

Reply:
yep, good catch

Reply:
I'd say leave it as it is and just use a comma separated list in the input